Bottom: 222c73fd22b6bf9b0713debadced5c50ae63472e
Top:    da2494a98d29e7246fb0219ceed9a9a1a54aa56d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 16:49:11 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 43bd1dc..f0d6724 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -246,8 +246,6 @@ ghid_draw_grid (BoxTypePtr drawn_area)
 
   hidgl_flush_triangles (&buffer);
 
-  glTexCoord2f (0., 0.);
-
   glDisable (GL_STENCIL_TEST);
   glEnable (GL_COLOR_LOGIC_OP);
   glLogicOp (GL_XOR);
